### PR TITLE
feature: load test

### PR DESCRIPTION
**Pull request by fireridlle**
### Notes
[Load test](https://faunadb.atlassian.net/browse/DRV-562)
Before release, we need to check if the driver can perform under production load usage.
Create multiple clients and run multiple concurrent queries

```
npm run load-test -- --help

Options:
      --help     Show help                                             [boolean]
      --version  Show version number                                   [boolean]
  -t, --threads  Maximum number of consequence threads    [number] [default: 10]
  -q, --queries  Maximum number of queries per each thread [number] [default: 500]
```

### CODE REVIEW

**Review by tigger9flow — APPROVED** [review]
LGTM
Please, don't forget to mention this script in the release instruction (confluence)

File: tools/loadTest.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+const getRandomElement = list => list[getRandomInt(0, list.length - 1)]
+
+const makeClient = () =>
+  new Client({
+    secret: testConfig.auth,
+    domain: testConfig.domain,
+    scheme: testConfig.scheme,
+  })
```
Comment (evbruno) [inline tools/loadTest.js:34]:
```suggestion
    port: testConfig.port
  })
```

Comment (evbruno) [reply tools/loadTest.js:34]: I had to add the port to run the `load-test` locally (inside a docker container); 
Is it supposed to catch the `FAUNA_PORT` (env var) from other place?

Comment (evbruno) [reply tools/loadTest.js:34]: In the end I got a few `(node:171) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 connect listeners added. Use emitter.setMaxListeners() to increase limit` but I think the PR is good to go

Comment (fireridlle) [reply tools/loadTest.js:34]: the approach of getting config is the same as used to be. driver tries to read file `testConfig` at the root dir. If failed, he tries to get config from env

**Review by evbruno — APPROVED** [review]

**Review by parkhomenko — APPROVED** [review]

**Pull request merged**